perfkitbenchmarker/linux_packages/mongodb_server.py

```diff
@@ -18,6 +18,7 @@
 
 def YumInstall(vm):
   """Installs the mongodb package on the VM."""
+  vm.RemoteCommand('sudo setenforce 0')
```

Comment:
Reviewing MongoDB's Documentation for running MongoDB Community Edition on RHEL/CentOS, the best option we currently have is to disable SELinux.  This is due to the fact PerfKit Benchmarker runs MongoDB on the data disk partition rather than the default mongodb directories (/var/lib/mongo, /var/log/mongodb) which requires more configuration of the SELinux policy.

- https://docs.mongodb.org/manual/tutorial/install-mongodb-on-red-hat/#configure-selinux

Additionally, we already disable SELinux when we install MySql on RHEL/CentOS distributions. (https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/linux_packages/mysql.py#L24)